src/components/common/LogisterButton.tsx

```diff
@@ -0,0 +1,60 @@
+import { Pressable, StyleSheet, Text, View } from 'react-native'
+
+interface LogisterButtonProps {
+    children: string
+    onPress: () => void
+    icon?: any
+    bgColor?: string
+    textStyle: any
+}
+
+function LogisterButton({ children, onPress, textStyle, bgColor, icon }: LogisterButtonProps) {
+    return (
+        <View style={styles.buttonOuterContainer}>
+            <Pressable
+                onPress={onPress}
+                style={
+                    ({ pressed }) => pressed
+                        ? [styles.pressed, styles.buttonInnerContainer, { backgroundColor: bgColor }]
```

Comment:
add the backgroundColor to the pressed style

Reply:
it is already added in the end of the array, isn't it?

Reply:
yes, but we prefer not to use inline style.
you can add this to the pressed style object